### ISSUE TITLE
How to change the port

### ISSUE THREAD

**Issue by omsenjalia** [mnutt/davros#142]
1. Hey my host only supports localhost:#### port is there a way to change port of the storage server? It will be very helpful if there is a way to do it...
2. So the servers store in sandstorm but I didn't install it cuz it didn't support my aarch64 is there a way to install it cuz my files are still there on my storage server it works without errors and without sand storm is there a solution/comment on this?

**Comment by ocdtrekkie** [comment]
Sandstorm doesn't presently support ARM systems. While it's something we'd like to do, it'd require a lot more work than Sandstorm is presently able to accomplish.

Davros also isn't currently built to work as a standalone app at this time. I don't think it's on mnutt's roadmap. It probably could be set up to run as such, but you'd be missing any sort of authentication at minimum.

**Comment by omsenjalia** [comment]
> Sandstorm doesn't presently support ARM systems. While it's something we'd like to do, it'd require a lot more work than Sandstorm is presently able to accomplish.
> 
> Davros also isn't currently built to work as a standalone app at this time. I don't think it's on mnutt's roadmap. It probably could be set up to run as such, but you'd be missing any sort of authentication at minimum.

I am kinda confused on what you said, as I am indian I don't have LOT of English knowledge could you explain this in more bit of text btw what's the answer to my question No.1

**Comment by omsenjalia** [comment]
> It probably could be set up to run as such, but you'd be missing any sort of authentication at minimum.

are you talking about changing ports? If yes I don't really care about authentication as I am the only one using it

**Comment by omsenjalia** [comment]
Another question: what's the difference if I install/dont install sandstorm

**Comment by omsenjalia** [comment]
Another question: how do I change thee Storage server URL to go from http://localhost:3009 to http://dono-01.danbot.host:1276 because that's my host's IP is there any way to do it like changing settings/file/code or a 3rd party software or something?

**Comment by ocdtrekkie** [comment]
In the readme it states how you can set it to run on a particular port: https://github.com/mnutt/davros#development

However, the application wouldn't care about the network name, localhost is just what you would use if you were accessing it from the same machine. Though as noted in the readme, Davros is not designed to be used without Sandstorm.

**Comment by omsenjalia** [comment]
It's not running on the same machine I am using a hosting called DBH it's a Pterodactyl panel hosting that's why I asked is there a way to use custom URL, because my host only supports http://dono-01.danbot.host:1276/ (1276 is the port of MY server in DBH)

**Comment by omsenjalia** [comment]
If I just change the port to 1276 and install sandstorm will it be possible to access the storage using http://dono-01.danbot.host:1276/files

**Comment by ocdtrekkie** [comment]
No, sorry, Sandstorm would not work on such a hosting provider. I don't think Davros would be a good solution for you.

**Comment by omsenjalia** [comment]
> No, sorry, Sandstorm would not work on such a hosting provider. I don't think Davros would be a good solution for you.

Is there any other Storage Server you know??

**Comment by mnutt** [comment]
Davros relies on Sandstorm for its authentication, so without using it via Sandstorm (or putting Davros behind some other sort of authentication mechanism, involves integration) then Davros would be open to the world and anyone could see/edit your files.
